test/server/cards/03_TWI/units/CaptainRexLeadByExample.spec.ts

```diff
@@ -0,0 +1,29 @@
+describe('Captain Rex Lead by Example', function() {
+    integration(function(contextRef) {
+        it('Captain Rex\'s ability should should create 3 Clone Tropper tokens when played', function () {
+            contextRef.setupTest({
+                phase: 'action',
+                player1: {
+                    hand: ['captain-rex#lead-by-example'],
+
+                },
+                player2: {
+                    groundArena: ['battlefield-marine']
+                }
+            });
+
+            const { context } = contextRef;
+
+            context.player1.clickCard(context.captainRex);
+
+            const cloneTroopers = context.player1.findCardsByName('clone-trooper');
+            expect(cloneTroopers.length).toBe(2);
+            expect(cloneTroopers).toAllBeInZone('groundArena');
+            expect(cloneTroopers.every((cloneTrooper) => cloneTrooper.exhausted)).toBeTrue();
+
+            // No Clone Trooper for player 2
+            const player2ArenaCard = context.player2.getArenaCards();
```

Comment:
If it's okay I would prefer that we test using the same pattern as the tests for Drop In. That lets us confirm that the right token type was created and that they came in exhausted, since both of those are potential mistakes that could have been made when writing the card implementation.

We can obviously see that you didn't make either of those mistakes, but most of the point of these tests is to catch typos in the implementation so we want to use the same test pattern every time if possible. Happy to discuss further if you have additional thoughts or suggestions.

Reply:
Thanks @AMMayberry1 . Very helpful to have directly pointer like Drop In for the test pattern to follow.